api/Clipboard.json

```diff
@@ -144,8 +136,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false,
-              "notes": "Firefox only supports reading the clipboard in browser extensions, using the <code>\"clipboardRead\"</code> extension permission."
+              "version_added": "preview"
```

Comment:
I believe this was actually behind `dom.events.asyncClipboard.readText` (which replaced `dom.events.asyncClipboard.read`) pref. This has comment _Control whether `navigator.clipboard.readText()` is exposed to content._ I'm confirming, but I think this means that the feature is enabled.

Reply:
Confirmed it is enabled. Now trying to work out correct note text.